### PR TITLE
Add bytecode mapping for Java 13

### LINKED ISSUE #80: Enforce Bytecode Version Rule does not support Java 13
The enforce-bytecode rule does not currently support Java 13. When it is executed it returns an error:

`Execution enforce-bytecode-version of goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce failed: Unknown JDK version given. Should be something like "1.7", "8", "11", "12" -> [Help 1]`

### PR DESCRIPTION
**Pull request by LukasPrediger**
Fixes #80

### CODE REVIEW

**Comment by khmarbaise** [comment]
You compile issues in your PR...

**Comment by LukasPrediger** [comment]
I had a few issues with the repository and my ide, so i switched to a plain editor. Fixed the issue and made sure the test execute properly

**Comment by athimel** [comment]
Travis failure does not seem to be related to the current PR

**Comment by sbrunov** [comment]
Hi, everyone!

How may we proceed with the pull request?  

Best regards,  
Sergey Brunov.

File: src/main/java/org/apache/maven/plugins/enforcer/EnforceBytecodeVersion.java [hunk 1]
```diff
@@ -95,6 +95,9 @@
         // Java 12
         JDK_TO_MAJOR_VERSION_NUMBER_MAPPING.put( "12", 56 );
         JDK_TO_MAJOR_VERSION_NUMBER_MAPPING.put( "1.12", 56 );
+
+	JDK_TO_MAJOR_VERSION_NUMBER_MAPPING.put( "13", 57);
+	JDK_TO_MAJOR_VERSION_NUMBER_MAPPING.put( "1.13", 57 );
```
Comment (sbrunov) [inline src/main/java/org/apache/maven/plugins/enforcer/EnforceBytecodeVersion.java:100]:
```suggestion
        JDK_TO_MAJOR_VERSION_NUMBER_MAPPING.put( "13", 57);
        JDK_TO_MAJOR_VERSION_NUMBER_MAPPING.put( "1.13", 57 );
```

**Comment by hazendaz** [comment]
At this point we need 14 and 15 too.  Since this doesn't amount to more than what looks like adding the mappings, can we just do a larger PR and add 14, 15 and even 16 for good measure so we are set for a year and also get it out quickly?

**Comment by mfriedenhagen** [comment]
I close this as I just merged #93 today.

**Pull request closed without merging**